CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 0.62.0
+Release 2018-mm-dd
+  - Add support for batch street-level geocoding analysis.
+
 ## 0.61.12
 Release 2018-mm-dd
```

Comment:
Hey `0.61.12` was not released, the last one was `0.61.11` which is the one with release date defined. You need to update this section (the title changes to 0.62.0).